<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 27, 36);
selection-background-color: rgb(38, 162, 105);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00.000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="accuracyLabel_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Время</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00.000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="accuracyLabel_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Слов в минуту</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Tilda">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>`</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_0">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Minus">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Equals">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>230</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Q">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_W">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_R">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_T">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Y">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_U">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_I">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_O">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_P">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_leftSquare">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_RightSquare">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_BackSlash">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>310</y>
      <width>61</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_J">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_K">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_L">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_UpKov">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>'</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_S">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_H">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_D">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_F">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_TochkaZap">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_A">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_G">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_V">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Dot">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_M">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_X">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Zap">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_B">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_N">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Slash">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_C">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Z">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Space">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>501</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_E">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QComboBox" name="languageChangeComboBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Французский</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Немецкий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Арабский</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Белорусский</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Иврит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Китайский</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="openTestButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть задание</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracyLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Точность</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>1450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TestText</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
